project 1/index.html

```diff
@@ -0,0 +1,110 @@
+            biologists
+            doing field research about birds and I had no idea what field research actually looked like and so the
+            scientists in
+            my novel draft did things like shuffle around great stacks of papers and frown. The good people of the
+            Earthwatch
+            organization assured me I was welcome on the trip and would get to participate in “real science” during my
+            time
+            on
+            the gulf. But as I waited to be picked up by my team in Corpus Christi, I was nervous—I imagined everyone
+            else
+            would
+            be a scientist or a birder and have daunting binoculars.</p>
+
+        <p>The biologist running the trip rolled up in in a large white van with a boat hitch and the words
+            <!--I used the bold tag because it is a 'product name' or the name of the truck.-->
+            <b>BIOLOGICAL
+                SCIENCES</b>
+            stenciled across the side. Jeff was forty-ish, and wore sunglasses and a backward baseball cap. He had a
+            winter
+            beard and a neon-green cast on his left arm. He’d broken his arm playing hockey with his sons a week before.
+            The
+            first thing Jeff said was, “We’ll head back to camp, but I hope you don’t mind we run by the liquor store
+            first.” I
+            felt more optimistic about my suitability for science.</p>
+
+        <div class="star">*</div>
+
+        <p>Not long before I’d called off my engagement it was Christmas.</p>
+
+        <p>The woman who was supposed to be my mother-in-law was a wildly talented quilter and made stockings with
+            Beatrix
+            Potter characters on them for every family member. The previous Christmas she had asked me what character I
+            wanted
+            to be (my fiancé was Benjamin Bunny). I agonized over the decision. It felt important, like whichever
+            character
+            I
+            chose would represent my role in this new family. I chose Squirrel Nutkin, a squirrel with a blazing red
+            tail—an
+            epic, adventuresome figure who ultimately loses his tail as the price for his daring and pride.</p>
+
+        <blockquote>
+            "I found a mouse. A mouse in a pink dress and apron. A mouse holding a broom and dustpan, serious about
+            sweeping."
+        </blockquote>
+
+        <p>I arrived in Ohio that Christmas and looked to the banister to see where my squirrel had found his place.
+            Instead, I
+            found a mouse. A mouse in a pink dress and apron. A mouse holding a broom and dustpan, serious about
+            sweeping. A
+            mouse named Hunca Munca. The woman who was supposed to become my mother-in-law said, “I was going to do the
+            squirrel
+            <!--I used the em tag because this part of the sentence is the fictional character's thoughts.-->
```

Comment:
For fictional character's thoughts, you want `i` (see here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i).

But since the mother-in-law is stating her thoughts, I wouldn't argue for the use of `i` and instead keep `em` for emphasizing what the she is saying, e.g. 

`<em>This</em> is CJ.`